<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="login_bottom" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item alignment="Qt::AlignLeft|Qt::AlignBottom">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>注册账号</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>304</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>304</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QComboBox" name="account_cb">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>298</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>298</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>298</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
	font-size: 20px;
	border: none;
	border-bottom:1px solid lightgray;
	background-color: transparent;
}

QComboBox:hover {
	border-bottom: 1px solid gray;
}
QComboBox:focus {
	border-bottom: 1px solid rgb(18, 183, 245);
}
QComboBox::drop-down {
	border-color: transparent;
	width: 60px;
	height:40px;
}
QComboBox:down-arrow {
	image: url(:/login/images/login_down_icon.png);
	width: 60px;
	height: 20px;
}

QComboBox QAbstractItemView {
	min-height: 60px;
}
QComboBox QAbstractItemView:item {
	color: lightblue;
}</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>990409144</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1600923552</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="pwd_line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>298</width>
           <height>35</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>298</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>298</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	font-size: 20px;
	border: none;
	border-bottom: 1px solid lightgray;
	background-color: transparent;
}
QLineEdit:hover{
	border-bottom: 1px solid gray;
}
QLineEdit:focus{
	border-bottom: 1px solid rgb(18, 183, 245);
}

</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="login_btn">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>200</y>
           <width>200</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(33, 174, 250);
	border-radius: 8px;
	color: white;
	spacing: 20px;
}
QPushButton:hover {
    background-color: rgb(72, 203, 250);
}
QPushButton:pressed {
    background-color: rgb(85, 85, 250);
}
</string>
         </property>
         <property name="text">
          <string>安全登录</string>
         </property>
         <property name="icon">
          <iconset resource="../images_res.qrc">
           <normaloff>:/login/images/login_icon.png</normaloff>:/login/images/login_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
        <widget class="QCheckBox" name="auto_login_cb">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>160</y>
           <width>87</width>
           <height>19</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>87</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>87</width>
           <height>19</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>自动登录</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="remember_pwd_cb">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>87</width>
           <height>19</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>87</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>87</width>
           <height>19</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>记住密码</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>301</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Matura MT Script Capitals</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="text">
          <string>Convolutional Neural Network</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignBottom">
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/login/images/QR_code.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/login/images/CNN.jpeg);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>show_register_pane()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>-17</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>open_cnn_link()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>account_cb</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>Login</receiver>
   <slot>enable_login_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwd_line</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Login</receiver>
   <slot>enable_login_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>login_btn</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>check_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remember_pwd_cb</sender>
   <signal>clicked(bool)</signal>
   <receiver>Login</receiver>
   <slot>auto_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>-47</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_login_cb</sender>
   <signal>clicked(bool)</signal>
   <receiver>Login</receiver>
   <slot>remember_pwd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_register_pane()</slot>
  <slot>open_cnn_link()</slot>
  <slot>enable_login_btn()</slot>
  <slot>check_login()</slot>
  <slot>auto_login()</slot>
  <slot>remember_pwd()</slot>
 </slots>
</ui>
